msg/YoloDetectedObject.msg

```diff
@@ -0,0 +1,6 @@
+string label
+float32 score
+int32 x0
+int32 y0
+int32 x1
+int32 y1
```

Comment:
I suggest to change the `x0`, `y0`, `x1`, and `y1` into more descriptive name (e.g. `top`, `left`, `bottom`, and `right`).
Also, change the `YoloDetectedObject` name to be more general (e.g. `DetectedObject`, `BoxDetection`, etc.).